feature-group-definitions/contain-intrinsic-size.yml

```diff
@@ -1,4 +1,5 @@
 name: contain-intrinsic-size
+description: The `contain-intrinsic-size` CSS property sets the size of an element that the browser will use for layout calculations when the element is subject to size containment, within a CSS query container.
```

Comment:
@vmpstr can you help review this description?

Reply:
This description seems good to me, so I'll just go ahead and merge this PR, and ask for review after the fact via a chat message.